packages/mds-ingest-service/tests/index.spec.ts

```diff
@@ -691,6 +691,24 @@ describe('Ingest Service Tests', () => {
     })
   })
 
+  describe.only('getLatestTelemetryForDevices', () => {
```

Comment:
I think we wanna remove the `.only` so that the other suites can run too!

Reply:
derp.